lib/improver/tests/acceptance/test_neighbour_finding.py

```diff
@@ -143,11 +155,13 @@ def test_alternative_coordinate_system(tmp_path):
                   "globe": {"semimajor_axis": 6378137.0,
                             "semiminor_axis": 6356752.314140356}}
     coord_opts_json = json.dumps(coord_opts)
-    args = [sites_path, orography_path, landmask_path, output_path,
-            "--site_coordinate_system", "LambertAzimuthalEqualArea",
-            "--site_coordinate_options", coord_opts_json,
-            "--site_x_coordinate", "projection_x_coordinate",
-            "--site_y_coordinate", "projection_y_coordinate"]
+    args = [orography_path, landmask_path,
+            "--site-list", sites_path,
+            "--site-coordinate-system", "LambertAzimuthalEqualArea",
+            "--site-coordinate-options", coord_opts_json,
+            "--site-x-coordinate", "projection_x_coordinate",
+            "--site-y-coordinate", "projection_y_coordinate",
+            "--output", output_path]
```

Comment:
The test data relating to this was fixed, but this test has not been updated. Please remove the xfail decorator and use a standard compare at the bottom:
```
acc.compare(output_path, kgo_path)
```

Reply:
I'll do this in a separate little PR.